### ISSUE TITLE
Minor fixes on cycling and walking profiles

### ISSUE THREAD

**Issue by OpenSourceForEver** [osmandapp/Osmand#6981]
Suggestions for cycling profile:
It is not needed "allow motorways" because bicycles are not allowed on motorways, instead you could include "allow private access".
It is not needed "Show alerts for speed cameras" and "Announce speed cameras" because speed cameras do not make sense for bicycles.

Suggestions for walking profile:
It is not needed "Show alerts for speed cameras", "Announce speed cameras" and "announce speed limit" because speed cameras and speed limits do not make sense for pedestrians.
It is not needed "avoid motorways" because pedestrians are not allowed on motorways. 
It is needed "avoid border crossings" (driving and cycling have this option).

**Comment by sonora** [comment]
At least bicycles **are** allowed on some motorways in some regions.

**Comment by OpenSourceForEver** [comment]
And what about the other improvements?

**Comment by vshcherb** [comment]
With new feature now we support custom routing profiles, so now we hope to get different views and probably start getting community building routing profiles for a specific region

**Comment by Themanwithoutaplan** [comment]
> At least bicycles **are** allowed on some motorways in some regions.

True, experienced this last year in Sweden south of Söderköping, when we the road we wanted to use was blocked due to road works. That said, it's not an experience I'd like to repeat!

**Comment by OpenSourceForEver** [comment]
https://github.com/osmandapp/Osmand/issues/6981#issuecomment-495971835
 
Ok, but what about walking profile? (avoid motorways, announce and show alerts for speed limits and speed cams)

**Comment by OpenSourceForEver** [comment]
> At least bicycles **are** allowed on some motorways in some regions.

Ok, but what about walking profile? (avoid motorways, announce and show alerts for speed limits and speed cams)
